dipy/segment/tests/test_clustering.py

```diff
@@ -445,20 +471,28 @@ def test_cluster_map_get_small_and_large_clusters():
     cluster_map = ClusterMap()
 
     # Randomly generate small clusters
-    indices = [rng.randint(0, 10, size=i) for i in range(1, nb_clusters+1)]
+    indices = [rng.randint(0, 10, size=i) for i in range(1, nb_clusters + 1)]
     small_clusters = [Cluster(indices=indices[i]) for i in range(nb_clusters)]
     cluster_map.add_cluster(*small_clusters)
 
     # Randomly generate small clusters
-    indices = [rng.randint(0, 10, size=i) for i in range(nb_clusters+1, 2*nb_clusters+1)]
+    indices = [rng.randint(0,10,size=i) 
+                for i in range(nb_clusters +1, 2*nb_clusters+1)]
     large_clusters = [Cluster(indices=indices[i]) for i in range(nb_clusters)]
     cluster_map.add_cluster(*large_clusters)
 
-    assert_equal(len(cluster_map), 2*nb_clusters)
-    assert_equal(len(cluster_map.get_small_clusters(nb_clusters)), len(small_clusters))
-    assert_arrays_equal(cluster_map.get_small_clusters(nb_clusters), small_clusters)
-    assert_equal(len(cluster_map.get_large_clusters(nb_clusters+1)), len(large_clusters))
-    assert_arrays_equal(cluster_map.get_large_clusters(nb_clusters+1), large_clusters)
+    assert_equal(len(cluster_map), 2 * nb_clusters)
+    assert_equal(len(cluster_map.get_small_clusters(
+        nb_clusters)), len(small_clusters))
+    assert_arrays_equal(
+        cluster_map.get_small_clusters(nb_clusters),
+        small_clusters)
+    assert_equal(len(cluster_map.get_large_clusters(
+        nb_clusters + 1)), len(large_clusters))
+    assert_arrays_equal(
+        cluster_map.get_large_clusters(
+            nb_clusters + 1),
+        large_clusters)
```

Comment:
This is a good place to break:

assert_equal(len(cluster_map.get_small_clusters(nb_clusters)),
                      len(small_clusters))